### ISSUE TITLE
no chunks produced

### ISSUE THREAD

**Issue by iManu** [BlessCSS/bless-brunch#7]
Hello,

Ihope you can help, I have installed bless-brunch but brunch seems to not produce chunked css files… 

The count result is 4,179.
When I use the cli manually to chunk, it's working.

I remove clean-css as recommended, and npm up brunch & dependencies…

When I run `brunch watch -d`for debug, the plugin loaded line not mention bless-brunch : 
`brunch:plugins Loaded plugins: eslint-brunch, fb-flo-brunch, less-brunch, postcss-brunch +857ms`

Here is the exports.config.plugins section of my brunch-config.js
 `
…
plugins: {
    autoReload : {
      enabled : {
        css : false,
        js: false,
        assets : false
      }
    },
    bless: {
        cacheBuster: true,
        cleanup: true,
        compress: true,
        force: true,
        imports: true,
        outputDirectory: './web/css/' // Output directory for blessed css files.
    },
    postcss: {
      processors: [
        require('autoprefixer')(['last 2 versions', 'ie >= 9', 'and_chr >= 2.3']),
        require('csswring')({
          preserveHacks: true
        }),
        require('postcss-url')({
          url: 'inline',
          filter: '**/*.svg',
          maxSize: 5
        }),
      ]
    }
  },
…
`
… and devDependencies in package.json

`
"devDependencies": {
    "autoprefixer": "^6.3.7",
    "bless-brunch": "^1.6.1",
    "brunch": "^2.8.2",
    "caniuse-db": "^1.0.30000613",
    "css-mqpacker": "^5.0.1",
    "csswring": "^5.1.0",
    "eslint-brunch": "^3.12.0",
    "eslint-config-airbnb-base": "^11.1.3",
    "eslint-plugin-import": "^2.2.0",
    "fb-flo-brunch": "^1.7.22",
    "install": "^0.8.4",
    "javascript-brunch": "^2.0.0",
    "less-brunch": "^2.6.0",
    "npm": "^4.1.1",
    "postcss-brunch": "^0.5.0",
    "postcss-merge-longhand": "^2.0.2",
    "postcss-merge-rules": "^2.1.1",
    "postcss-url": "^5.1.2",
    "uglify-js-brunch": "^2.0.1"
  }
`

**Comment by thomasconner** [comment]
@iManu Can you setup a sample project for me to take a look? The tests for the plugin produce the chunks that would be expected.

**Comment by thomasconner** [comment]
@iManu I created a simple sample project. I installed `brunch@2.10.9` and `bless-brunch@2.0.0`. When I run `brunch build`, I see the chunks output to the build directory specified. Closing because I am not able to reproduce.

**Comment by thomasconner** [comment]
I have included the example project in the repo. Please take a look at https://github.com/BlessCSS/bless-brunch/tree/master/example

**Comment by iManu** [comment]
@thomasconner Thank you for the speed 

I made a sample here https://github.com/iManu/sample-bless

Project is "cleaned" and the problem persist…  Blessc count said 4,181. Cli chunk also work.

You will find all node modules that are used in my project. Brunch file is similar i just remove useless files paths.

Regards

ps: I tried to install bless-brunch 2.0.0 but failed with relative and/or git methods in package.json. Target is not yet available on npm.
Also tried with Brunch 2.10.9 updated but still the same.
